src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RANDOM_VALUE = 100;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private static final int DEFAULT_RADIUS = 10;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure generateFigure() {
+        switch (FigureType.values()[random.nextInt(FigureType.values().length)]) {
+            case CIRCLE:
+                return getRandomCircle();
+            case SQUARE:
+                return getRandomSquare();
+            case RIGHT_TRIANGLE:
+                return getRandomRightTriangle();
+            case ISOSCELES_TRAPEZOID:
+                return getRandomIsoscelesTrapezoid();
+            case RECTANGLE:
+                return getRandomRectangle();
+            default:
+                throw new RuntimeException("Figure does not exist");
```

Comment:
We should use the default method
```suggestion
                return getDefaultFigure();
```

Reply:
Thanks, I'll fix it🤝